src/component/icon.tsx

```diff
@@ -1,21 +1,12 @@
 import * as React from 'react';
-import { style } from 'typestyle';
 
-const iconitem = style({
-    backgroundRepeat: 'no-repeat',
-    backgroundSize: '24px',
-    backgroundPositionX: '0px',
-    backgroundPositionY: '5px',
-    minHeight: '28px',
-    minWidth: '35px',
-    display: 'inline-block'
-});
+import { iconitem } from './style/icon';
 
-export interface IProps {
+export interface IIconProps {
```

Comment:
Usually when designating an `IProps` interface that belongs to another object you can make a `namespace` instead. 

```
export namesapce IconItem {
    export interface IProps {
        source: string;
    }
}
```